packages/seacas/libraries/ioss/src/main/io_shell_ts.C

```diff
@@ -170,32 +178,15 @@ int main(int argc, char *argv[])
 #ifdef SEACAS_HAVE_MPI
   MPI_Init(&argc, &argv);
   MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+  ON_BLOCK_EXIT(MPI_Finalize);
 #endif
 
   std::cout.imbue(std::locale(std::locale(), new my_numpunct));
```

Comment:
See above comment on `new my_numpunct`.

Reply:
The memory is reported as "still reachable" by valgrind:
```
==39946== 144 bytes in 1 blocks are still reachable in loss record 35 of 58
==39946==    at 0x1000A3FA4: malloc (vg_replace_malloc.c:302)
==39946==    by 0x1000C69AB: operator new(unsigned long) (in /opt/local/lib/libgcc/libstdc++.6.dylib)
==39946==    by 0x1000017A5: main (test.C:12)
```
(This is from a simple test program reproducing the above snippet)